media-api/app/lib/ImageResponse.scala

```diff
@@ -24,18 +25,37 @@ object ImageResponse {
 
   def fileMetaDataUri(id: String) = URI.create(s"${Config.rootUri}/images/$id/fileMetadata")
 
+  def hasPersistenceIdentifier(image: Image) =
+    image.identifiers.contains(Config.persistenceIdentifier)
+
+  def hasExports(image: Image) =
+    image.exports.nonEmpty
+
+  def isArchived(image: Image) =
+    image.userMetadata.exists(_.archived)
+
   def isPhotographerCategory[T <: UsageRights](usageRights: T) =
     usageRights match {
       case _:Photographer => true
       case _ => false
     }
 
-  // TODO: move this as a method of Image (fiddly due to dep on Config)
-  def imageIsPersisted(image: Image) = {
-    image.identifiers.contains(Config.persistenceIdentifier) ||
-      image.exports.nonEmpty ||
-      image.userMetadata.exists(_.archived) ||
-      isPhotographerCategory(image.usageRights)
+  def imagePersistenceState(image: Image): (Boolean, List[String]) = {
+    val reasons: ListBuffer[String] = ListBuffer[String]()
+
+    if (hasPersistenceIdentifier(image))
+      reasons += "persistence-identifier"
+
+    if (hasExports(image))
+      reasons += "exports"
+
+    if (isArchived(image))
+      reasons += "archived"
+
+    if (isPhotographerCategory(image.usageRights))
+      reasons += "photographer-category"
+
+    (reasons.nonEmpty, reasons.toList)
```

Comment:
Why not just return the reasons list and then do the check [here](https://github.com/guardian/grid/pull/1259/files#r39658832)?